src/reconcile/rec_write.c

```diff
@@ -3286,6 +3285,17 @@ __rec_split_write(WT_SESSION_IMPL *session,
 		}
 	}
 
+	bnd->entries = r->entries;
+	/* Output a verbose message if we create a page without many entries */
+	if (WT_VERBOSE_ISSET(session, WT_VERB_SPLIT) && r->entries < 6)
+		WT_ERR(__wt_verbose(session, WT_VERB_SPLIT,
+		    "Reconciliation creating a page with %" PRIu32
+		    " entries, memory footprint %" PRIu64
+		    ", page count %" PRIu32 ", %s, split state: %d\n",
+		    r->entries, r->page->memory_footprint, r->bnd_next,
+		    F_ISSET(r, WT_EVICTING) ? "evict" : "checkpoint",
+		    r->bnd_state));
+
```

Comment:
@keithbostic I realized I didn't reply to your question here. You are right that the entry count isn't an ideal metric, but it was a good filter for the workload I was using, and I'm happy for that to evolve over time.
Do you run test/format with verbose split turned on very often? I've thought a couple of times that it would be nice if verbose messages had a unique identifier, to make it simple to filter out just the messages you are interested in keeping.